src/WPF/ArcGISRuntime.WPF.Samples/Samples/Geometry/FormatCoordinates/FormatCoordinates.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.FormatCoordinates.FormatCoordinates"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013"
+             mc:Ignorable="d"
+             d:DesignHeight="300" d:DesignWidth="300">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+        <Label Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2" Foreground="Red">Type in the textboxes to see a real-time conversion</Label>
+        <Label Grid.Row="1" Grid.Column="0" HorizontalAlignment="Right">Decimal Degrees</Label>
+        <TextBox Grid.Row="1" Grid.Column="1" Tag="Decimal Degrees" x:Name="DecimalDegreesTextField" />
+        <Label Grid.Row="2" Grid.Column="0" HorizontalAlignment="Right">Degrees, Minutes, Seconds</Label>
+        <TextBox Grid.Row="2" Grid.Column="1" Tag="Degrees, Minutes, Seconds" x:Name="DmsTextField" />
+        <Label Grid.Row="3" Grid.Column="0" HorizontalAlignment="Right">UTM</Label>
+        <TextBox Grid.Row="3" Grid.Column="1" Tag="UTM" x:Name="UtmTextField" />
```

Comment:
Would be cool to implement this using data binding on the text fields and a value converter that converts the coordinate text to the correct format. (Just an observation, not a suggestion ;))